public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/comment-meta.php

```diff
@@ -363,3 +364,37 @@ function get_feedback_questions( $version = META_VERSION ) {
 
 	return $questions;
 }
+
+/**
+ * Count feedback comments marked as helpful for a specific post or overall.
+ *
+ * @param int $post_id
+ *
+ * @return int
+ */
+function count_helpful_feedback( $post_id = 0 ) {
+	$post__in = array();
+	if ( $post_id ) {
+		$post__in[] = absint( $post_id );
+	}
+
+	$meta_query = array(
+		array(
+			'key'   => 'helpful',
+			'value' => 1,
+			'type'  => 'NUMERIC',
+		),
+	);
+
+	$feedbacks = get_feedback( $post__in, array( 'hold', 'approve' ), $meta_query );
```

Comment:
This function changed in the feedback UI PR to [accept all WP_Query args,](https://github.com/WordPress/wordcamp.org/blob/f977142e506a58b8a2561751b83d1793ac33ec03/public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/comment.php#L183) so this last param should be `array( 'meta_query' => $meta_query )`

Reply:
👍 ddb753ce